lib/stripe/accounts.ex

```diff
@@ -1,128 +1,122 @@
 defmodule Stripe.Accounts do
   @moduledoc """
-  Main API for working with Customers at Stripe. Through this API you can:
-  -create accounts
-  -get single account
-  -delete single account
-  -delete all account
-  -count account
-
-  https://stripe.com/docs/api/curl#account_object
+  Functions for working with customers at Stripe. Through this API you can:
+
+    * create accounts,
+    * get single account,
+    * delete single account,
+    * delete all account,
+    * count account.
+
+  Stripe API reference: https://stripe.com/docs/api/curl#account_object
   """
 
   @endpoint "accounts"
 
   @doc """
-  Creates an Account with the given parameters
+  Creates an account given the account params.
+
+  Returns created account.
 
   At the bare minimum, to create and connect to a managed account, simply set
   managed to true in the creation request, and provide a country. The country,
   once set, cannot be changed.
 
   https://stripe.com/docs/connect/managed-accounts
 
-  ## Example
-
-  ```
-    new_account = [
-      email: "test@example.com",
-      managed: true,
-      tos_acceptance: [
-        date: :os.system_time(:seconds),
-        ip: "127.0.0.1"
-      ],
-      legal_entity: [
-        type: "individual",
-        dob: [
-          day: 1,
-          month: 1,
-          year: 1991
+  ## Examples
+
+      params = [
```

Comment:
I'm using 4 spaces, as this seems to preferred way of styling the docs (per examples from Elixir codebase).